docs/calicoctl/bgp.md

```diff
@@ -0,0 +1,217 @@
+
+# User guide for 'calicoctl bgp' commands
+
+This sections describes the `calicoctl bgp` commands.
+
+These commands can be used to manage the global BGP configuration.  This 
+includes default values to use for the AS number, whether a full BGP mesh is
+required between all of the Calico nodes, and global BGP peers (these are BGP
+speakers that peer with every Calico node in the network).
+
+In addition to global BGP configuration, there is also Calico node specific
```

Comment:
Maybe snappier restructuring.  Suggestion, could start with something like "Calico node specific configuration, such as a peer specific to a particular Calico node, is configured using the 'calico node' blah blah..."